packages/teams-js/src/public/dialog.ts

```diff
@@ -114,7 +123,13 @@ export namespace dialog {
    */
   export function registerOnMessageFromParent(listener: PostMessageChannel): void {
     ensureInitialized(FrameContexts.task);
+    removeHandler('messageForChild');
```

Comment:
This needs to be removed when registering to receive parent messages because we will now send messages.

Reply:
Can you write a comment line in code about un-registering please? It would help others to understand

Reply:
Nit: please add a space" following "//".